android/src/main/kotlin/com/fluttercandies/photo_manager/core/utils/IDBUtils.kt

```diff
@@ -94,9 +94,10 @@ interface IDBUtils {
         size: Int,
         requestType: Int = 0,
         option: FilterOption,
+        forceGetLocation: Boolean
     ): List<AssetEntity>
 
-    fun getAssetEntity(context: Context, id: String): AssetEntity?
+    fun getAssetEntity(context: Context, id: String,forceGetLocation: Boolean = false): AssetEntity?
```

Comment:
`forceGetLocation` should be inside the `FilterOption`. Also, the code is not well-formatted.